artemis-ra/src/main/java/org/apache/activemq/artemis/ra/ActiveMQResourceAdapter.java

```diff
@@ -122,6 +122,10 @@
 
    private String entries;
 
+   //fix of JBEAP-12720 - propagated value of transactional attribute JMSConnectionFactoryDefinition annotation with the
```

Comment:
Should this reference an Apache jira?

Reply:
Hi, I've missed this reference, too, I'll change it to ARTEMIS-1669.